aim/sdk/index_manager.py

```diff
@@ -183,7 +183,7 @@ def index(
                 ).subtree('meta')
                 meta_run_tree = meta_tree.subtree('chunks').subtree(run_hash)
                 meta_run_tree.finalize(index=index)
-                if meta_run_tree['end_time'] is None:
+                if meta_run_tree.get('end_time') is None:
```

Comment:
This is a bug in the current aim code. I f there is no value for 'end_time', the code raises an exception instead of returning None.